### PR TITLE
Major restructure - tutorials, diagrams, fundamentals, FAQ, dev env etc

### PR DESCRIPTION
**Pull request by dan-glass**
Restructures the Space Services User Guide:

 1. Hierarchical
 2. Expands the fundamentals, getting started, execution environment etc.
 3. Adds FAQ
 4. Adds payload environment setup for python modules
 5. Adds many tutorials

Review is requested from product (Idris) & Marketing (Monika L) as well as @nsat/constpipe 

See https://developers.sbox.spire.com/space-services-user-guide/

Transcripts: 
 - [x] [hello_world](https://paste.spire.com/onatesezix.bash)
 - [x] [rf_transmit](https://paste.spire.com/posemuvava.http)
 - [x] [rf_collect](https://paste.spire.com/eyenoduyiz.makefile) ([spectrogram](https://spire-chat.slack.com/files/UP5RT4RGC/F03CCHNPMV3/20220421t192350z_rf_collect-int16_spectrogram.png))
 - [ ] [lease isl]()
 - [x] [ipc](https://paste.spire.com/isimoyocam.json)
 - [x] [copy](https://paste.spire.com/johuyadeyu.bash)
 - [x] [aoi]()
 - [x] [adcs]()
 - [x] [cuda](https://paste.spire.com/uronegivus.bash)
 - [x] [cudnn](https://paste.spire.com/ezefigekoh.bash)

### CODE REVIEW

File: AttitudeControl.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Attitude is controlled by the Attitude Control and Determination System (ADCS). Attitude information can be set as part of a window, or at run-time, in-orbit when there is an ADCS lease window, via the [Satellite Bus API](https://developers.spire.com/satellite-bus-api/). The ADCS provides a way for the satellite to reorient itself, and report which way it is facing. Attitude information is always relative to an aperture/antenna. If none is provided then the X/S-BAND radio is assumed.
+
+No other controls (e.g. thrusters or orbit changes) are available to users. [Orbital station keeping](https://en.wikipedia.org/wiki/Orbital_station-keeping) is performed by Spire.
+
+
+## Setting ADCS on windows
+
+If the user knows ahead of time what the are of interest is (e.g. tracking a point on earth or facing a fixed direction) then this information can be provided in [`adcs_config`](https://developers.spire.com/tasking-api-docs/#adcs_config) on the  payload window. For example, to take a photo of a location on earth, the user would schedule an [`PAYLOAD_IPI`](https://developers.spire.com/tasking-api-docs/#payload_ipi) window to have the the `IPI` aperture track that location:
```
Comment (spire-carlos) [inline AttitudeControl.md:10]: Nit:

```suggestion
Attitude is controlled by the Attitude Control and Determination System (ADCS).
Attitude information can be set as part of a window, or at run-time, in-orbit
when there is an ADCS lease window, via the [Satellite Bus API].
The ADCS provides a way for the satellite to reorient itself, and report which
way it is facing.
Attitude information is always relative to an aperture/antenna. If none is
provided then the X/S-BAND radio is assumed.

No other controls (e.g. thrusters or orbit changes) are available to users.
[Orbital station keeping] is performed by Spire.


[Satellite Bus API][https://developers.spire.com/satellite-bus-api/]
[Orbital station keeping][https://en.wikipedia.org/wiki/Orbital_station-keeping]

## Setting ADCS on windows

If the user knows ahead of time what the are of interest is (e.g. tracking a
point on earth or facing a fixed direction) then this information can be
provided in [adcs_config] on the  payload window. For example, to take a photo
of a location on earth, the user would schedule an [PAYLOAD_IPI] window to have
the the `IPI` aperture track that location:

[adcs_config][https://developers.spire.com/tasking-api-docs/#adcs_config]
[PAYLOAD_IPI][ https://developers.spire.com/tasking-api-docs/#payload_ipi ]

```
I think it makes it easier to maintain if we limit line length, and use reference links.

Comment (dan-glass) [reply AttitudeControl.md:10]: > limit line length
Everyones IDE is different. I use line-wrap to overcome this issue. 

> use reference links.
Assuming you mean relative links. This is not possible as the markdown is transformed by slate-docs into a website that is published to https://developers.sbox.spire.com/space-services-user-guide/index.html

**Comment by samanthawagner** [comment]
Some comments after reading the manual:

1. I would have the Open page as a General Space service, not as the SW in space page.
2. the first slide has a very big size / police VS the rest of the page
3. On Radios, L-Band is missing.
4. I think the 4 sections (Run code, perform exp, control and contact GS assets are missing, or at least I can't click on the chapters.
5. Payload in Space need more explanation / graphics. It is very short.
6. Fundamentals: a lot of "chapters" are in fact just definitions. I would do a separate definition chapter and skip those as they add noise.
7. I find it hard-ish to navigate, as most chapter actually are "just" a link to another page, and when you click on it you loose the page you were in. I would either have one full document, or make a default "open in another windows" for the links.
8. All explantions otherwise are easy to follow and well explained imo. But would be good to ask someone that is "spire space services" agnostic for feedback.

**Review by jordanano — APPROVED** [review]
LGTM as it applies to NICSAT - assuming this is the most up-to-date content and we've applied any lessons learned to it @dan-glass ?

**Pull request merged**